### PR TITLE
PMM localized labels - 238

### PR DESCRIPTION
**Pull request by dmkud**
# Critical Changes

# Changes

Updated translations and fr-ca files

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD|FLS|Tab|Classes
- [ ] Field sets are updated to account for new fields
- [ ] UX approval or UX not necessary
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

File: metadeploy/labels_fr-CA.json [hunk 1]
```diff
@@ -9,7 +9,11 @@
...
+<<<<<<< HEAD
+            "message": "Bienvenue dans le programme d’installation de Program Management Module (PMM)!\r\n\r\nProgram Management Module est un cadre standard pour assurer le suivi, la gestion et la livraison de programmes, de services et de livraison de services.\r\n\r\n*IMPORTANT* : Salesforce recommande de ne pas installer les mises à niveau directement dans les organisations de production. Au lieu de cela, installez d’abord Program Management Module dans l’un de vos environnements sandbox ou dans une organisation de développeurs.\r\n\r\n*Paragraphe légal si nécessaire*\r\n\r\nAprès avoir installé Program Management Module, consultez les ressources dans notre documentation :\r\n\r\n* <Documentation sur Program Management Module>\r\n\r\nVous avez des questions? Devenez membre du groupe de clients <Program Management Module> dans le hub Power of Us.",
```
Comment (ptewson-sfdo) [inline metadeploy/labels_fr-CA.json:13]: @dmkud It looks like this merge conflict made it in here ... any idea which of these two "message" attributes are the correct one?

File: metadeploy/labels_fr_CA.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+{
```
Comment (ptewson-sfdo) [inline metadeploy/labels_fr_CA.json:1]: @dmkud - this file may be an "extra", duplicating `labels_fr-CA.json`, but I don't want to remove it from this PR if there might be some updates we bring into that other file ... any thoughts on how to approach that?

**Comment by jjbennett** [comment]
Opened a [feature branch PR ](https://github.com/SalesforceFoundation/PMM/pull/601) and merged in changes from here.

**Comment by jjbennett** [comment]
Opened a [feature branch PR ](https://github.com/SalesforceFoundation/PMM/pull/601) and merged in changes from here.

**Pull request closed without merging**